src/game/game.c

```diff
@@ -13,6 +13,17 @@
 
 #include <SDL2/SDL_image.h>
 
+static void set_window_title(struct oshu_game **game)
```

Comment:
`**game` is needed when you wanna write constructors and use them like:

```
struct thing *t;
create(&t);
```

Here it only makes the object harder to manipulate.

Reply:
Usually, a setter function takes a value, not just the destination. What about renaming it to `update_window_title`?